openfisca_core/periods.py

```diff
@@ -358,6 +358,41 @@ def __str__(self):
             if fragment is not None
             )
 
+    def __cmp__(self, other):
+        unit_a, start_a, size_a = self
+        unit_b, start_b, size_b = other
+
+        if unit_a != unit_b:
+            unit_weights = {
+                u'day': 1,
```

Comment:
https://github.com/openfisca/openfisca-core/blob/master/openfisca_core/periods.py#L118

The question is "Do we really _use_ periods with `day` as a unit ?" I dare not check...